pandunia-loge.csv

```diff
@@ -3095,14 +3093,15 @@ sesam|sesam|biw|Sesamum indicum|tur:susam, eng:sesame, rus:сезам (sezam), s
 set.yom:|setyom||||Sunday|dimanche|domingo|domingo|воскресенье|يوم الأحد|星期日 (禮拜日)|日曜日|일요일|chủ nhật|रविवार|রবিবার|minggu|jumapili|pazar|dimanĉosunnuntai|niedziela|niedziela
 seti|seti|||spa:siete, por:sete, fra:sept, hin:सात (sāt), ben:সাত (sat), tha:เจ็ด (chet), yue:七 (chat1), jpn:七 (shichi), tur:yedi|seven (7)|sept (7)|siete (7)|sete (7)|семь (7)|七 (7)|七 (7)|||||||||sep (7)|seitsemän (7)|siedem (7)
 setomes|setomes|mese|||July||julio|||||７月|||||||||heinäkuu|październik
-Sexel nesia|Sexel nesia|desh|SC||Seychelles||Seychelles||||||||||||||Seychellit|Seszele
+Seyxel nesia|Seyxel nesia|desh|SC||Seychelles||Seychelles||||||||||||||Seychellit|Seszele
 siah:i|siahi|rang||fas:سیاه‎ (siyâh), hin:सियाह (siyāh), urd:سیاہ (siyāh), pan:ਸਿਆਹ (siāh), tur:siyah|black|noir|negro|preto|чёрный|أَسْوَد|黑色|黒い|검다|đen|काला|কালো|hitam|eusi|kara (siyah)|nigra|musta|czarny
 sian:i|siani|||eng:cyan, spa:cian, rus:циан (tsian), ara:سيان (sayan), hin:क्यान (kyāna), jpn:シアン (shian), may:sian|cyan||cian||циан|||シアン|||||||||syaani|niebieskozielony (cyjan)
 Sibir:ia|Sibiria||||Siberia||Siberia||||||||||||||Siperia|Syberia
 siborg.em:|seaborgem|mate|Sg||seaborgium||seaborgio|seabórgio|сиборгий|||シーボーギウム|||||seaborgium|seaborgi|siborgium|seborgio||seaborg
 sida|sida|||rus:сидеть (sidet'), eng:sit, deu:sitzen, ita:sedere|sit (put down)||sentar||сидеть|||||||||||sidi|istua|siedzieć
 side|side||||seat (saddle)||silla (montura)||сиденье (седло)||座部 (马鞍)|座席 (鞍)||||||||||siedzenie (siodło)
 sidu|sidu||||sit down||sentarse|||||座る||||||||||usiąść
+Siera: Leon:|Siera Leon|desh|SL||Sierra Leone||Sierra Leona||||||||||||||Sierra Leone|Sierra Leone
```

Comment:
nam sa "Salone" va krio bax. ("Krio is the most widely spoken language across Sierra Leone, spoken by 97% of the country's population.")